taskcluster/ac_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,172 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+from mozilla_version.maven import MavenVersion
```

Comment:
Bug:

> This probably means we need to install the mozilla-version dependency before we run the decision task. I'm not seeing where we do this in Fenix. This may be a question for @JohanLorenzo :)

Ah yeah, this something we hit on Fenix. We had to install the `mozilla-version` package in the decision task at these 2 places: 

* https://github.com/mozilla-mobile/fenix/blob/2cada405d81fe4831e866794c91e57fb088315ac/.taskcluster.yml#L264
* https://github.com/mozilla-mobile/fenix/blob/2cada405d81fe4831e866794c91e57fb088315ac/.taskcluster.yml#L270

We also have to bump the docker image defined in `.taskcluster.yml`: 

https://github.com/mozilla-mobile/fenix/commit/7d3b23ceeb12c0e3edcb36836e9041b9ca7a1800#diff-a728f7e52d751b98eafa856e45594533339b44f229d7b83f930df335391e7e15R246

The reasons is not super explicit, but let me explain:

 * The decision task uses a docker image that already contains some python packages, and notably `attrs`
 * `mozilla-version` requires an up-to-date version of `attrs`
 * on the current docker image, attrs is too old and does not get updated. See line 64 of `live_backing.log` https://firefox-ci-tc.services.mozilla.com/tasks/fM6I_bXkQeeANWIBKquWsA
 * the new docker image has a compatible `attrs`


@escapewindow, in the long term, we likely want to bake this package in https://hg.mozilla.org/ci/taskgraph/file/2b2622598df02bde211d8cedb334b7b22fb883a4/taskcluster/docker/decision-mobile/Dockerfile

Reply:
ok, I've done this.